Sources/WeaverDI/ThreadSafeBuilderStoringDecorator.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ThreadSafeBuilderStoringDecorator.swift
+//  WeaverDIPackageDescription
+//
+//  Created by Bruno Mazzo on 5/31/18.
+//
+
+import Foundation
+
+class ThreadSafeBuilderStoringDecorator: BuilderStoring {
+    
+    let builderStoring: BuilderStoring
+    private let queue: DispatchQueue = DispatchQueue(label: "ThreadSafeBuilderStoringDecorator")
```

Comment:
Let's use the type description for the label of the dispatch queue.
```swift
DispatchQueue(label: "\(ThreadSafeBuilderStoringDecorator.self)")
```

Reply:
Done